<launch>

	<group ns="pdt_module">
		<node pkg="pdt_module" type="objects_detection" name="objects_detection_node" args="-c $(find pdt_module)/src/objects_detection/cvpr2012_very_fast_over_bahnhof.config.ini">
			<!-- <remap from="topic_name" to="topic_name" /> -->
			<param name="ConfigFile" value="new_file_name" />
		</node>
		<!-- <node pkg="turtlesim" type="ground_estimation" name="ground_estimation_node" args="-c ConfigFile.txt" /> -->
	</group>

</launch>
